index.html

```diff
@@ -106,6 +106,7 @@ <h5 class="nav-category">
     require('./assets/demo-btns')
     require('./assets/code-blocks')
     require('./assets/normalize-shortcuts')
+    require('./assets/devtron')
```

Comment:
Feels a little weird to ship this script with the app and run it in all cases since `devtron` is a dev dependency and it will never be included in the distribution.

There is a `-r` option to the Electron CLI to preload modules. I wonder if we could use through `npm start` to install Devtron when it is done that way.